src/main/java/org/alliancegenome/curation_api/services/validation/dto/fms/HTPExpressionDatasetAnnotationFmsDTOValidator.java

```diff
@@ -118,13 +122,53 @@
 
 		if (CollectionUtils.isNotEmpty(dto.getCategoryTags())) {
 			List<VocabularyTerm> categoryTags = new ArrayList<>();
+			ObjectMapper mapper = new ObjectMapper();
 			for (String categoryTag : dto.getCategoryTags()) {
 				if (StringUtils.isNotEmpty(categoryTag)) {
-					VocabularyTerm tag = vocabularyTermService.getTermInVocabulary(VocabularyConstants.HTP_DATASET_CATEGORY_TAGS, categoryTag).getEntity();
-					if (tag == null) {
-						htpAnnotationResponse.addErrorMessage("categoryTags", ValidationConstants.INVALID_MESSAGE + " (" + categoryTag + ")");
+					String query = """
+						{
+							"searchFilters": {
+								"vocabularyNameFilter": {
+									"vocabulary.name": {
+										"queryString": "%s",
+										"useKeywordFields": true,
+										"queryType": "matchQuery"
+									}
+								},
+								"synonymsOrNameFilter": {
+									"synonyms": {
+										"queryString": "%s",
+										"useKeywordFields": true,
+										"queryType": "matchQuery"
+									},
+									"name": {
+										"queryString": "%s",
+										"useKeywordFields": true,
+										"queryType": "matchQuery"
+									}
+								}
+							}
+						}
+					""".formatted("Data Set Category Tags", categoryTag, categoryTag);
+					Map<String, Object> params = new HashMap<>();
+					try{
+						params = mapper.readValue(query, new TypeReference<Map<String, Object>>() {});
+					} catch (Exception e) {
+						e.printStackTrace();
+						break;
+					}
+					SearchResponse<VocabularyTerm> searchResponse = vocabularyTermService.searchByParams(new Pagination(), params);
+					if(searchResponse.getTotalResults() == 1) {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' is not followed by whitespace.
<!-- __reviewdog__:ChBmOGJiYzdlZTU2NzJiNGQ3EgpjaGVja3N0eWxl -->